File: College_Assignments/LAB-3/prime.c
//Program to print all Prime numbers between 100 and 200

#include <stdio.h>

int main()
{
    int i, j, check;
    for (i = 100; i <= 200; i++)
    {
        check = 0;
        for (j = 2; j < i / 2; j++)
        {
            if (i % j == 0)
            {
                check++;
                break;
            }
        }
        if (check == 0)
        {
            printf("%d\t", i);
        }
    }
    printf("\n");
    return 0;
}

File: Dotnet (C#)/Lab 1/prime.cs
// WAP to print prime numbers from 100 to 200

using System;

class prime {
    static void Main(string[] args) {
        Console.WriteLine("Prime numbers between 100 and 200:");
        for (int num = 100; num <= 200; num++) {
            bool isPrime = true;

            for (int i = 2; i <= Math.Sqrt(num); i++) {
                if (num % i == 0) {
                    isPrime = false;
                    break;
                }
            }

            if (isPrime && num > 1) {
                Console.WriteLine(num);
            }
        }
    }
}
